src/main/java/mate/controller/car/AllCarController.java

```diff
@@ -0,0 +1,23 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+public class AllCarController extends HttpServlet {
```

Comment:
Use informative names for your controllers. Always start controller’s name with a verb: GetAllManufacturersController, AddDriverToCarController etc.

https://mate-academy.github.io/jv-program-common-mistakes/java-web/web-practice/java-web-practice